persistent/Database/Persist/TH.hs

```diff
@@ -273,6 +287,211 @@ preprocessUnboundDefs preexistingEntities unboundDefs =
     (embedEntityMap, noCycleEnts) =
         embedEntityDefsMap preexistingEntities unboundDefs
 
+-- | Include entities that are related to each other in one call
+derivePersist :: MkPersistSettings -> PersistSettings -> [DeriveEntityDef] -> Q [Dec]
```

Comment:
Before the version 2.13 taking a list of entities was mandatory if they were referencing each other. Now it is just a design choice for API. It may be worth to merge this with `mkDeriveEntityDef` but it's not clear whether it's more ergonomic.